src/appleseed-max-impl/utilities.cpp

```diff
@@ -171,18 +171,23 @@ std::string insert_texture_and_instance(
     }
 
     const std::string texture_name = wide_to_utf8(bitmap_tex->GetName());
-    base_group.textures().insert(
-        asr::DiskTexture2dFactory::static_create(
+    if (base_group.textures().get_by_name(texture_name.c_str()) == nullptr)
```

Comment:
This will make renderer use one map even if max's scene contains other maps with the same name. With this change renderer will keep rendering and not disappearing like it does now.